mc/Voyage-Mongo-Tests.package/VOMongoTest.class/instance/waitForWriteOperation.st

```diff
@@ -0,0 +1,5 @@
+running
+waitForWriteOperation
+	"Write operations can be performed in a fire-and-forget mode, and it's convenient to wait a bit to ensure mongodb performed it before asserting on the results."
+
+	50 milliSeconds wait
```

Comment:
Again you can use >>#command: with getLastError and force a sync? Then the _last_ write/delete/op must have succeeded.